commit eaeed6c85c9fa29b10a0ff4557ed6177f34c6c73 (refs/changes/89/5489/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-02-05T16:52:41-08:00 (8 months ago)
    
    MANTA-4092 Binder loses race with Zookeeper on startup leading to detached zk session watchers

diff --git a/package.json b/package.json
index 61a9dca..943a519 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "vasync": "2.1.0",
     "verror": "^1.10.0",
     "xtend": "1.0.3",
-    "zkstream": "0.11.0"
+    "zkstream": "0.11.3"
   },
   "engines": {
     "node": ">=0.10"
